e2etests/cloud/run_cloud_uimode_test.go

```diff
@@ -318,7 +318,7 @@ func TestCloudSyncUIMode(t *testing.T) {
 					want: RunExpected{
 						Status: 1,
 						StderrRegexes: []string{
-							string(clitest.CloudLoginRequiredMessage),
+							string(clitest.CloudOnboardingIncompleteMessage),
```

Comment:
all of those tests were actually for incomplete onboarding because it fakes a 404 in the `/v1/users` endpoint, which means user is authenticated but didn't signup.